packages/tasks/src/model-libraries.ts

```diff
@@ -769,6 +769,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/TensorSpeech/TensorFlowTTS",
 		snippets: snippets.tensorflowtts,
 	},
+	"tabpfn": {
+		prettyLabel: "TabPFN",
+		repoName: "TabPFN",
+		repoUrl: "https://github.com/PriorLabs/TabPFN",
+		filter: false,
+		countDownloads: `path_extension:"ckpt"`,
```

Comment:
```suggestion
		countDownloads: `path_extension:"ckpt" or path:"config.json"`,
```

maybe?

Reply:
@merveenoyan @pcuenca  Sorry bringing this back up on a closed PR but doing this will double-count downloads since both the ckpt and the config.json files are downloaded (or attempted to be downloaded) in the download process. See https://github.com/PriorLabs/TabPFN/blob/faf092b49b255f0e68270947eeeca4817b0013d4/src/tabpfn/model/loading.py#L143. What I would do is to remove this line which will default to the default counting rule which is "count on config.json downloads".

Reply:
opened https://github.com/huggingface/huggingface.js/pull/1100